### DISCUSSION TITLE
5573: Probleme nach Firmware Update auf 3.05

### DISCUSSION THREAD

**Discussion by mattt3** [Tom-Bom-badil/samson_trovis_557x#53] [General (de/en)]
Hallo,

ich habe eine Trovis 5573.
Hardwareversion 131
Anlage 4.1

Firmwareversion neu: 3.05
Firmwareversion alt: 2.32

Die Trovis kann ich mit der seitlichen Buchse ansprechen, sowohl über USB-Adapter als auch mit EW10A Seriell-Wifi Adapter.

Ich habe mittels Bootmanager und USB ein Firmwareupdate gemacht.
Das lief erfolgreich durch. Die neue Version wird angezeigt und steht im Register.
Nach dem Update musste ich die alte Konfiguration (vor dem Update gesichert) wieder aufspielen, weil die Trovis nur noch RK1 auf dem Display angezeigt hat und nichts weiter gemacht hat.

Alle Werte habe ich mit Trovis-View ausgelesen und verglichen, stimmen überein.
In Trovis-View V4.89 kann ich nur maximal V3.02 auswählen, habe ich erst zu spät gesehen. Auslesen funktioniert, Trovis-View konvertiert die Version.

Problem ist jetzt: Sobald die Speicherladepumpe einschaltet, weil die Speichertemperatur gefallen ist, regelt Trovis RK1 auf 0, macht also primär den Vorlauf zu.
Wenn ich Register "Vorlaufsollwert Rk1 (CL116)" anschaue, ändert sich beim Einschalten der SLP der Sollwert auf 5°C.
Im Handbetrieb kann ich die Ventile öffnen, dann wird auch der Speicher geladen.

Habt ihr Ideen zur Behebung? Warum wird der RK1 Sollwert auf 5°C gesetzt?

**Comment by calle2010** [comment]
Mit dem Problem kann ich Dir leider nicht helfen, habe nur eine einfache Anlage 1.0. Ich benutze allerdings auch einen Elfin EW10A und habe damit kein Firmware-Update hinbekommen (siehe https://github.com/Tom-Bom-badil/samson_trovis_557x/discussions/6#discussioncomment-11135637). Es wäre toll, wenn Du die Details Deiner Konfiguration auch in den Thread eintragen könntest. Evtl. gibt es noch Einstellungen, die ich anders machen muss. Oder es liegt doch nur an der bescheidenen Wifi-Empfangsstärke im Heizungsraum.

**Reply by mattt3** [reply]
Über EW10A klappt das Update bei mir auch nicht, Trovis zeigt Boot es geht aber nicht weiter.
Habe das Update über USB/Serial gemacht

**Reply by Tom-Bom-badil** [reply]
Das Problem mit den Firmware-Updates habe ich auch schon häufiger beobachten können. Irgendwie wird bei manchen Adapter die Verbindung nach dem obligatorischen Reboot der Trovis zurückgesetzt.

Kurioserweise kann ich zB mit dem TCP-E2 im seriellen Modus (COMx: = RTU über virtuellen seriellen Port) komplett erfolgreich updaten, während derselbe Adapter im Modbus TCP-Modus nach dem Reboot von der Flash-Software nicht mehr gefunden und das bereits hochgeladene Update somit nicht angewendet wird. Vielleicht hat das etwas mit unterschiedlichen Timeouts bei COMx: Emulation (=Software/Treiber hält Verbindung) im Vergleich zur Direktverbindung per Modbus TCP zu tun, wo die Flash-Software die Verbindung direkt zur Ziel-TCP aufbaut. Ist aber nur geraten.

/tom

**Comment by Tom-Bom-badil** [comment]
> weil die Trovis nur noch RK1 auf dem Display angezeigt hat und nichts weiter gemacht hat.

Das ist typisch, wenn die Reglereinstellungen zurückgesetzt wurden. "Nur Rk1" ist Anlage 1.0, die Werkseinstellung. Das würde auch erklären, warum Du die Einstellungen erneut hochladen musstest.

>Habt ihr Ideen zur Behebung? Warum wird der RK1 Sollwert auf 5°C gesetzt?

Leider nicht. Anlage 4.1 läuft auf einer Trovis in einer Pewo-Anlage in der näheren Verwandtschaft. Wenn ich mich richtig erinnere, gibt es da einen gemischten und einen ungemischten Kreis hinter dem Wärmetauscher, die per separaten Pumpen gespeist werden (zuzüglich WW). Erwarten würde ich eigentlich, dass für WW das Stellventil in Rk1 (=primär) auf 100% gefahren wird (Werkseinstellung) und die beiden Pumpen für Hk1 und Hk2 abgeschaltet werden, um nicht die hohen WW-Vorlauftemperaturen in die Heizkreise zu geben. Dafür wird dann die SLP eingeschaltet. Dass er stattdessen vorn zumacht, ist wirklich komisch ...

/tom

**Reply by mattt3** [reply]
OK, danke für die Info bezüglich zurücksetzen.

Mit der alten Firmware wurde RK1 je nach Bedarf aufgefahren. Die SLP wurde eingeschaltet, die Pumpe für RK2 (Fussbodenheizung) ausgeschaltet.

Pumpen schalten auch noch, bloss der Sollwert passt nicht. Bei 5° Soll und VF1 von über 50° macht der Regler dann RK1 zu. Ich vermute dass irgendein Register einen anderen Wert braucht in der neuen FW.

Firmware 2.32
![FW232](https://github.com/user-attachments/assets/0bf982d3-1c09-432c-a0e2-adb9bd7cb42a)
Firmware 3.05
![FW305](https://github.com/user-attachments/assets/bb634825-efcc-4d90-b6c7-5555fbe1bb21)
gegen 8:00 habe ich hier wieder manuell RK1 und RK2 geöffnet

**Reply by calle2010** [reply]
Ich habe nochmal im Handbuch gestöbert. Bei der Vorrangschaltung Inversregelung ist beschrieben, wie der Heizkreis auf bis zu 5°C Vorlauf runtergeregelt wird. Aber eigentlich nur, wenn die Trinkwassererwärmung primärseitig erfolgt. Der Parameter CO4 F08 ist aber auch für Anlage 4.1 dokumentiert. 
Ansonsten gibt es ja auch Telefonsupport von Samson oder evtl. Support durch Deinen FW-Anbieter.

**Reply by Tom-Bom-badil** [reply]
Ich verstehe den Plot schon bei Firmware 2.32 nicht. Das Stellventil für WW wird grad mal auf knapp unter 40% aufgefahren - dafür der Mischer im Heizkreis auf 60%? Unabhängig davon, dass Rk1 auf 100% hochgehen müsste - hast Du Flächenheizung oder Heizkörper (letzteres könnte zumindest den hohen Wert in Rk2 erklären).

/tom

**Reply by mattt3** [reply]
> Ich verstehe den Plot schon bei Firmware 2.32 nicht. Das Stellventil für WW wird grad mal auf knapp unter 40% aufgefahren - dafür der Mischer im Heizkreis auf 60%? Unabhängig davon, dass Rk1 auf 100% hochgehen müsste - hast Du Flächenheizung oder Heizkörper (letzteres könnte zumindest den hohen Wert in Rk2 erklären).
> 
> /tom

Guter Hinweis. Da war die Ventillaufzeit Ty zu groß, habe auf den Default 35s zurückgestellt. Ich habe Flächenheizung im ganzen Haus.

**Comment by Nesselbrand67** [comment]
Ein downgrade auf Firmware Version 2.32 ist nicht möglich? Wenn es da wieder funktioniert würde ich erstmal das bleiben bis zu der v3.05 von Samson im Releaselog die Änderungen beschrieben sind

**Reply by Nesselbrand67** [reply]
Ich sehe gerade, dass Du ja einen recht großen Versionssprung gemacht hast.
Gab es in der Zwischenzeit vielleicht für dich relevante Änderungen?
Aus dem ChangeLog der 5573-1 sehe ich z.B.:
2.51: Bedeutung von CO1, CO2 -> F02 geändert F02 - 1 = Witterungsgeführte Regelung aktiv -> nicht dass dein AF abgeschaltet ist?
2.60: Freie Analog-Ausgangszuweisung: unter CO5 -> F34 wird festgelegt, welches Ausgangssignal am 0-bis-10-V-Ausgang ausgegeben werden soll. -> nicht das Dein RK1 Ventil ein 10V Steuersignal benötigt und nun nicht mehr bekommt?

**Reply by mattt3** [reply]
danke für den Hinweis mit dem changelog. Wenn ich die Handbücher für 2.32 und 3.05 vergleiche, finde ich:
CO1 > F02 - 1 mit AF1. das ist auch so konfiguriert.
neu ist wohl CO2 > F02 - 1 mit AF1. Den Wert gab es in der 2.32 noch nicht.

Das mit dem Analogausgang schließe ich aus, da ich das Ventil RK1 (und RK2) manuell auffahren kann.

**Reply by mattt3** [reply]
> Ein downgrade auf Firmware Version 2.32 ist nicht möglich? Wenn es da wieder funktioniert würde ich erstmal das bleiben bis zu der v3.05 von Samson im Releaselog die Änderungen beschrieben sind

die alte Firmware wird leider nicht zum Download bei Samson angeboten

**Comment by Nesselbrand67** [comment]
@mattt3 gibt es von deiner Seite Neuigkeiten zur Version 3.05? 
Hast du eine Lösung gefunden?

**Reply by mattt3** [reply]
Hallo,
Lösung habe ich noch keine, nur einen Workaround.

Habe jetzt gesehen, dass der Sollwert RK1 bereits auf 5°C reduziert wird, wenn das Warmwasser abkühlt und den Sollwert 55°C erreicht. Sobald SF1 auf 54,9°C fällt, wird die RK1 Pumpe ab- und die SLP eingeschaltet, das funktioniert.

Die Vorrangschaltung Inversregelung habe ich probehalber eingeschaltet, bessert das Verhalten aber nicht.

Workaround ist, dass ich per Automatisierung den Stellwert RK1 aufmache, wenn das Wasser dann warm ist schalte ich wieder auf autark und lasse den Regler normal die FBH regeln.

**Reply by mattt3** [reply]
Habe die Konfigurationen 2.32 (linke Hälfte im Vergleich) und 3.05 (rechte Hälfte) als .xml exportiert und verglichen.
[2.32 vs 3.05](https://www.textcompare.org/xml/?id=674b40281e087b10dd5a8552)
Falls jemand Lust hat :-) das anzuschauen

**Reply by Nesselbrand67** [reply]
Habe ich zumindest nicht. Was mich aber stutzig macht sind die doch vielen Änderungen.

Macht Trovis-View vielleicht Anpassungen an den xml Files, wenn die Version sich ändert und damit u.U. die xml Syntax?
Dann vielleicht warten bis Trovis-View die 3.05 unterstützt oder erstmal auf eine supportete Version downgraden?

Hast Du versucht, nach Werksreset mit einer manuellen Konfiguration am Gerät die Anlage 4.1 zum Laufen zu bringen?

**Reply by mattt3** [reply]
ja, ich warte auf eine aktualisierte TrovisView. Die Idee mit dem Werksreset behalte ich im Hinterkopf, könnte klappen.
Falls jemand eine Firmware <3.05 archiviert hat und mir zukommen lässt, kann ich das versuchen

**Comment by mattt3** [comment]
Update: Es gibt eine neue Firmware 3.06. Nach dem Installieren ist das Problem behoben.
Der Sollwert RK1 geht nicht mehr auf 5°C, sondern auf 65°C (55 TW Soll + 10 Überhöhung).
